### PR TITLE
Fix (dactionary copy/paste)

### PR DESCRIPTION
**Pull request by Aca-jov**

### CODE REVIEW

**Comment by ioanpocol** [comment]
:+1:

File: client/app/scripts/superdesk/editor/spellcheck/spellcheck.js [hunk 1]
```diff
@@ -118,21 +119,27 @@ function SpellcheckService($q, api, dictionaries, editor) {
     /**
      * Find errors in given text
      *
-     * @param {string} text
```
Comment (petrjasek) [inline client/app/scripts/superdesk/editor/spellcheck/spellcheck.js]: i think you can keep the param just change it to `{Node} node`

**Comment by petrjasek** [comment]
:+1:

File: client/app/scripts/superdesk/editor/spellcheck/spellcheck.js (hunk 1 above)
Comment (Aca-jov) [inline client/app/scripts/superdesk/editor/spellcheck/spellcheck.js]: Yes, I deleted it, forgot to return.. I returned it now..

**Pull request merged**